docs/README.md

```diff
@@ -291,19 +291,19 @@ Stackdriver Monitoring ([documentation](https://cloud.google.com/monitoring/docs
 
 #### Using Monitoring
 
-To get to Stackdriver Monitoring from the GCP console, select **Monitoring** on the navigation panel. This brings up the Stackdriver Monitoring console, a separate UI from the consoles for other GCP and Stackdriver products.  By default, you reach an overview page:
+To get to Stackdriver Monitoring from the GCP console, select **Monitoring** on the navigation panel. By default, you reach an overview page:
```

Comment:
It's not "Stackdriver Monitoring", it's "Cloud Monitoring". Please change throughout.

Reply:
Got it

Reply:
I put this change in #57 since we discussed doing a full replacement as a separate task from this.